docs/guides/src/docs/asciidoc/_includes/setting-up-the-sample-boot.asc

```diff
@@ -6,36 +6,40 @@ starter-appname: the name of the module users should start with to complete the
 starter-config-type: the type of configuration the starter sample uses: javaconfig, xml, boot
 completed-appname: the name of the module that contains the completed application
 completed-config-type: the type of configuration the completed sample uses: javaconfig, xml, boot
-download-url: the URL to download the Spring Security distribution
 
 ////
 
 == Setting up the sample
 
-This section outlines how to setup a workspace within https://spring.io/tools/sts[Spring Tool Suite (STS)] so that you can follow along with this guide. The next section outlines generic steps for how to apply Spring Security to your existing application. While you could simply apply the steps to your existing application, we encourage you to follow along with this guide in order to reduce the complexity.
+This section outlines how to setup a workspace within https://spring.io/tools[Eclipse IDE with Spring Tools] so that you can follow along with this guide. The next section outlines generic steps for how to apply Spring Security to your existing application. While you could simply apply the steps to your existing application, we encourage you to follow along with this guide in order to reduce the complexity.
 
 === Obtaining the sample project
 
-Extract the {download-url}[Spring Security Distribution] to a known location and remember it as _SPRING_SECURITY_HOME_.
+Clone the {gh-samples-url}[Spring Security Samples project] to a known location and remember it as _SPRING_SECURITY_HOME_.
+[NOTE]
+====
+These samples are being migrated to a separate project, however, you can still find
+the not migrated samples in the {gh-old-samples-url}[Spring Security repository].
```

Comment:
To clarify, I think we should say "the not migrated samples in an older branch of the Spring Security repository."